re-openssl-opt.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/mk/re.mk b/mk/re.mk
+index 788943f..d626a29 100644
+--- a/mk/re.mk
++++ b/mk/re.mk
+@@ -427,7 +427,8 @@ endif
+ # External libraries section
+ #
+ 
+-OPENSSL_OPT := $(shell [ -f /usr/local/opt/openssl/include/openssl/ssl.h ] \
++OPENSSL_OPT :=
```

Comment:
This should not be needed, you can call `make OPENSSL_OPT=` to override

Reply:
@sreimers thanks. Done